avocado/utils/multipath.py

```diff
@@ -106,6 +106,19 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "egrep -i mpathd /etc/multipath/bindings"
```

Comment:
using bash in python not looks good, Better to use re module to processing of file content, even I suggest use `genio` to populate file content

Reply:
Also, take in consideration other systems:

In the multipath documentation there is a configuration parameter called user_friendly_names with the following definition:

If set to yes, specifies that the system should use the /etc/multipath/bindings file to assign a persistent and unique alias to the multipath, in the form of mpath n. If set to no, specifies that the system should use the WWID as the alias for the multipath. In either case, what is specified here will be overridden by any device-specific aliases you specify in the multipaths section of the configuration file. The default value is no.

Reply:
Check removed on bindings, as the name can be overridden with the help of multipaths section in configuration file. Code is updated to find the WWID from the name (name can be user friendly name or the wwid itself) in any case the code will return the wwid of the device.

Reply:
ah still i see we are using bash any way we can target this as refactor  for whole multipath API in future